samples/testing-frameworks/flutter/README.md

```diff
@@ -130,3 +128,36 @@ These are scripts for creating test package:
 - zip-test-files-ios-simulator-install-flutter.sh
   - creates ios simulator integration test package (install flutter)
   - upload ipa not apk file with this
+
+  **Steps for Bitbar cloud iOS real devices runs**
```

Comment:
Should we advice users to try out iOS real device testing when it might not work?